docs/user/exams/instructors_guide.rst

```diff
@@ -36,9 +36,20 @@ During the exam creation and configuration, you can create your exam and configu
 
 - You can create an exam by clicking on |create_new_exam|.
 
-1.2 Create and Configure Exam
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-- When you click on |create_new_exam| you are presented with the *Create Exam* view. Here you can set the basic information such as ``title``, ``examiner`` etc. The :ref:`timeline of the exam <timeline>` is defined by the dates: ``visible from``, ``start of working time``, ``end of working time``, ``release date of results``, ``begin of student review``, ``end of student review``.
+1.2 Configure Exam
```

Comment:
There are three references in this file that link to the old section name by name
- line 25
- line 471
- line 500

Those references should be updated to use a proper `:ref:` to a defined id.
```suggestion
_configure_exam:

1.2 Configure Exam
```

The ref name for 1.2.1 Create Exam should then be renamed to avoid conflicting ids.